sql/bit.go

```diff
@@ -69,6 +69,11 @@ func MustCreateBitType(numOfBits uint8) BitType {
 	return bt
 }
 
+// MaxResponseByteLength implements Type interface
+func (t bitType) MaxResponseByteLength() uint32 {
+	return uint32(t.numOfBits)
```

Comment:
This should be a flat 8 bytes for an `uint64`, as we don't change the return type based on the bit size. Also, we'd have to align the bits to bytes if we did.

Reply:
Because the text response protocol returns one byte for each individual bit, I think this is the right implementation. Another data point is that MySQL returns `12` for the max field length for `bit(12)` columns. 

But... you still have a good point that this can be a little confusing to read. I'll at least add a comment to help make that clear and I might even try renaming the interface method to something like `MaxTextResponseByteLength`. That might help make it more self-explanatory that this is for a text (not binary) representation.